src/components/z-book-card/index.stories.mdx

```diff
@@ -38,7 +40,10 @@ const defaultArgTypes = {
 # ZBookCard
 
 <Canvas>
-  <Story name="All props">
+  <Story
+    name="All props"
+    args={{ribbonIcon: "info", ribbonInteractive: true}}
```

Comment:
Questa riga che è stata aggiunta non dovrebbe servire. Questa storia prende tutti gli args di default (come impostati alle linee 15-16), e mi sta bene che di default non ci sia l'icona né il ribbon interattivo. Poi l'utente è in grado di modificarli a suo piacere da interfaccia.